hsa-ext-rocr/PKGBUILD

```diff
@@ -1,29 +1,21 @@
 # Maintainer: acxz <akashpatel2008 at yahoo dot com>
 # Contributor: Bruno Filipe <bmilreu@gmail.com>
+# Contributor: Ranieri Althoff <ranisalt+aur at gmail.com>
+
 pkgname=hsa-ext-rocr
-pkgver=1.1.30100
+pkgver=3.1.0
 pkgrel=1
 _debfile=hsa-ext-rocr-dev_1.1.30100.0-rocm-rel-3.1-44-ecafeba1_amd64.deb
-pkgdesc="ROCm Platform Runtime: Closed source components"
+pkgdesc='ROCm Platform Runtime: Closed source components'
 arch=('x86_64')
-url="https://github.com/RadeonOpenCompute/ROCm"
+url='https://github.com/RadeonOpenCompute/ROCR-Runtime'
 license=('EULA')
-groups=()
-depends=(hsa-rocr hsakmt-roct)
-makedepends=()
-provides=("${pkgname%-git}")
-conflicts=("${pkgname%-git}")
-replaces=()
-backup=()
-options=()
-source=("http://repo.radeon.com/rocm/apt/debian/pool/main/h/hsa-ext-rocr-dev/${_debfile}")
+depends=("hsa-rocr>=$pkgver" "hsakmt-roct>=$pkgver")
```

Comment:
Is there a need for having versioned deps? Since we control the whole stack and only the latest version will be provided. I do not see a reason to add this. In general, versioned deps are looked down upon unless absolutely needed.

Reply:
I am unsure as to why AMD always releases the same version for everything and they are all the same, so I assume they must be tied together e.g. 3.0.0 would lack interfaces needed for 3.1.0.

In the case we control the stack it is not a problem though.

Reply:
kk we can worry about versioned deps in a later meta-issue if need be.